e2e/Makefile

```diff
@@ -199,10 +202,13 @@ daemonset-lvmd/setup-minikube:
 
 .PHONY: daemonset-lvmd/launch-minikube
 daemonset-lvmd/launch-minikube:
-	$(BINDIR)/minikube start \
+	$(SUDO) $(BINDIR)/minikube start \
 		--vm-driver=none \
 		--kubernetes-version=v$(KUBERNETES_VERSION) \
 		--extra-config=kubelet.read-only-port=10255
+	$(SUDO) chown -R $$USER $$HOME/.kube $(MINIKUBE_HOME)/.minikube
+	$(SUDO) chmod -R a+r /home/runner/.kube $(MINIKUBE_HOME)/.minikube
```

Comment:
this can be replaced with $$HOME.
https://github.com/topolvm/topolvm/pull/309/commits/0947a86cd8ac65a6cdb682c570a1cbe6e49a3f3e

Reply:
@toshipp Feel free to update it :-)

Reply:
Yes, I'll do that.